lib/gui/index.html

```diff
@@ -49,5 +50,15 @@
         manifest-bind="version"
         os-open-external="https://github.com/resin-io/etcher/blob/master/CHANGELOG.md"></span>
     </footer>
+
+    <div class="section-loader"
+      ng-controller="StateController as state"
+      ng-class="{
+        isFinish: state.is('success')
+      }">
+      <safe-webview
+        src="'https://etcher.io/success-banner/'"
```

Comment:
It looks like the `src` attribute of the `safe-webview` AngularJS directive is configured to expect an expression rather than a value. This allows the user to pass a variable that is defined in the controller, for example.

I'm happy with both approaches. Taking an interpolation rather than an expression would remove the quotes, but it wouldn't allow a variable, although we're not using that at the moment.